src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/DataAPiHelpers.kt

```diff
@@ -0,0 +1,63 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.integration
+
+import org.springframework.http.HttpHeaders
+import org.springframework.test.web.reactive.server.WebTestClient
+import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.controllers.DraftAdjudicationResponse
+import java.time.LocalDateTime
+import java.util.function.Consumer
+
+class DataAPiHelpers(private val webTestClient: WebTestClient, private val defaultHeaders: Consumer<HttpHeaders>) {
```

Comment:
Pull out API helpers methods so that we can use them in the reported adjudication integration tests